bendigobank.rb

```diff
@@ -2,69 +2,79 @@
 
 require 'watir'
 require 'nokogiri'
+require 'selenium-webdriver'
+require 'json'
 require 'pry'
 require_relative 'account'
 require_relative 'transaction'
 
-# Operate over a bank account.
+# class operates over a bank account.
 
 class Bendigobank
+
+
   def execute
-    login
-    fetch_accounts
-    fetch_transactions
-    show_result
+   connect
+   fetch_accounts
+   fetch_transactions
+    puts @accounts
+    close
   end
 
+  binding.pry
+  browser = Watir::Browser.new(:chrome)
+  browser.goto('https://demo.bendigobank.com.au')
+
   def login
-    # here you log in to the bank
-    browser = Watir::Browser.new(:chrome, 'goog:chromeOptions' => { detach: true })
-    browser.goto('https://demo.bendigobank.com.au')
-    # sign in script
-    browser.button(:tabindex => "4", :class => "input_submit",
-      :name => "customer_type",:value =>"personal").click
+    browser.button(:tabindex => "4", :class => "input_submit", :name => "customer_type",:value =>"personal").click
   end
 
   def fetch_accounts
-  # fetch html data using nokogiri, take only fragment of html.
-  html = Nokogiri::HTML.fragment(browser.div(class: 'accounts-list').html)
+    html = Nokogiri::HTML.fragment(browser.div(class: "content__wrapper").html)
   parse_accounts(html)
```

Comment:
Add 2 spaces for correct indentation.
Read again style guide https://rubystyle.guide/